### PR TITLE
R119 : deprecate Spark Enrich, Cloudfront collector, Clojure collector 

### PR DESCRIPTION
**Pull request by benjben**

### CODE REVIEW

File: 3-enrich/README.md [hunk 1]
```diff
@@ -10,13 +10,15 @@ The **Enrich** process takes raw Snowplow events logged by a [Collector][collect
...
 | [scala-common-enrich][e4]      | A shared library for processing raw Snowplow events, used in (1) and (3) | Production-ready |
 | [emr-etl-runner][e5]           | A Ruby app for running (1) and (2) on Amazon Elastic MapReduce           | Production-ready |
 
 ## How to add an enrichment?
 
+**TO BE UPDATED**
+
 5 things to update:
 1. [iglu-central](https://github.com/snowplow/iglu-central): holds the JSON schema(s) required for the enrichments.
 2. [scala-common-enrich](./scala-common-enrich/): library containing all the enrichments. This library is then used in enrichment jobs like [spark-enrich](./spark-enrich/), [stream-enrich](./stream-enrich/) or [beam-enrich](./beam-enrich/).
```
Comment (oguzhanunlu) [inline 3-enrich/README.md:24]: `spark-enrich` is still here, I see `TO BE UPDATED` above, is it to remove spark-enrich from this line or anything else?

Comment (benjben) [reply 3-enrich/README.md:24]: It's still not clear to me what to do about `spark-enrich`. The plan is to move it to `snowplow/spark-enrich` and keep merging OSS PRs, so I think that it should still be mentionned. But I agree that it's a bit weird to mention a component that we consider deprecated. @chuwy any thoughts on that, should we stop mentioning `spark-enrich` ?

`TO BE UPDATED` is for how to add an enrichment tutorial that is in the README.

Comment (chuwy) [reply 3-enrich/README.md:24]: Forgot to raise this question, sorry. I think we should:

1. Extract it to `snowplow/spark-enrich`
2. Mark it as "not actively maintained"
3. Accept PRs whereas possible
4. _Generally_ should stop mention it as a recommended engine, but maybe list somewhere

I think it still has a lot of value.

File: 3-enrich/spark-enrich/README.md [hunk 2]
```diff
@@ -1,67 +1,6 @@
 # Spark Enrich
 
-## Introduction
+As introduced in [R118 blogpost](https://snowplowanalytics.com/blog/2020/01/16/snowplow-release-r118-badrows/) and previously in the [RFC about the deprecation of batch](https://discourse.snowplowanalytics.com/t/rfc-making-the-snowplow-pipeline-real-time-end-to-end-and-deprecating-support-for-batch-processing-modules/3018), we no longer support Spark Enrich.
```
Comment (chuwy) [inline 3-enrich/spark-enrich/README.md:3]: I'd delete the whole directory.

File: 3-enrich/README.md [hunk 3]
```diff
@@ -10,13 +10,15 @@ The **Enrich** process takes raw Snowplow events logged by a [Collector][collect
...
 |---------------------------------|--------------------------------------------------------------------------|------------------|
-| [spark-enrich][e1] (1)  | The Snowplow Enrichment process built using Apache Spark   | Production-ready |
+| [spark-enrich][e1] (1)  | The Snowplow Enrichment process built using Apache Spark   | Deprecated |
 | [stream-enrich][e3] (3)        | The Snowplow Enrichment process built as an Amazon Kinesis application   | Production-ready |
 | [scala-common-enrich][e4]      | A shared library for processing raw Snowplow events, used in (1) and (3) | Production-ready |
 | [emr-etl-runner][e5]           | A Ruby app for running (1) and (2) on Amazon Elastic MapReduce           | Production-ready |
 
 ## How to add an enrichment?
 
+**TO BE UPDATED**
```
[the remark this answers is not in the document]
Comment (benjben) [reply 3-enrich/README.md:20]: Following R118 refactoring this `How to add an enrichment?` tutorial is deprecated. I will just remove it from the README, we can always write an updated one later.

File: 3-enrich/README.md (hunk 1 above)
Comment (benjben) [reply 3-enrich/README.md]: I agree that `spark-enrich` has still a lot of value. I updated the link [here](https://github.com/snowplow/snowplow/tree/r118-morgantina/3-enrich#available-enrich) to `snowplow/spark-enrich` and updated its status to "Not actively maintained" as you proposed.

**Pull request merged**